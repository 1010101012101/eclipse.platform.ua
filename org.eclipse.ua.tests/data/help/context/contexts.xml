<?xml version="1.0" encoding="UTF-8"?>
<?NLS TYPE="org.eclipse.help.contexts"?>

<contexts>
   <context id="test.id" >
      <description>This is a test context.</description>
	  <topic label="This topic should always appear" href="data/help/context/simple_page.html"/>
	  
	  <!-- should never appear -->
	  <topic filter="os=invalid" label="Your OS is invalid" href="data/help/context/simple_page.html"/>
	  <topic filter="ws=invalid" label="Your WS is invalid" href="data/help/context/simple_page.html"/>
	  <topic filter="arch=invalid" label="Your ARCH is invalid" href="data/help/context/simple_page.html"/>
	  <topic label="Your OS is win32 AND your WS is invalid" href="data/help/context/simple_page.html">
	    <filter name="os" value="win32"/>
        <filter name="ws" value="invalid"/>
	  </topic>

	  <topic filter="plugin=org.eclipse.help" label="The plugin org.eclipse.help is installed" href="data/help/context/simple_page.html"/>
	  <topic filter="plugin!=org.eclipse.help" label="The plugin org.eclipse.help is NOT installed" href="data/help/context/simple_page.html"/>

      <!-- should never appear -->
      <topic filter="plugin=my.invalid.plugin.id.12345" label="The plugin my.invalid.plugin.id.12345 is installed" href="data/help/context/simple_page.html"/>

      <topic filter="plugin!=my.invalid.plugin.id.12345d" label="The plugin my.invalid.plugin.id.12345 is NOT installed" href="data/help/context/simple_page.html"/>
      <topic filter="product=org.eclipse.sdk.ide" label="The product org.eclipse.sdk.ide is running" href="data/help/context/simple_page.html"/>

      <!-- should never appear -->
      <topic filter="product=invalid" label="The product invalid is running" href="data/help/context/simple_page.html"/>

      <topic filter="category=org.eclipse.categories.developmentCategory" label="The category org.eclipse.categories.developmentCategory is enabled" href="data/help/context/simple_page.html"/>

      <!-- should never appear -->
      <topic filter="category=invalid" label="The category invalid is enabled" href="data/help/context/simple_page.html"/>

      <topic filter="activity=org.eclipse.javaDevelopment" label="The activity org.eclipse.javaDevelopment is enabled" href="data/help/context/simple_page.html"/>

      <!-- should never appear -->
      <topic filter="activity=invalid" label="The activity invalid is enabled" href="data/help/context/simple_page.html"/>	
	  
   </context>
</contexts>
