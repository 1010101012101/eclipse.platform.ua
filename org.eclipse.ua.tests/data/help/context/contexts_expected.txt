<?xml version="1.0" encoding="UTF-8" ?>
<contexts>
   <context id="test.id">
      <description>This is a test context.<description>
      <topic
            label="This topic should always appear"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
      <topic
            label="The plugin org.eclipse.help is installed"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
      <topic
            label="The plugin my.invalid.plugin.id.12345 is NOT installed"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
      <topic
            label="The product org.eclipse.sdk.ide is running"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
      <topic
            label="The category org.eclipse.categories.developmentCategory is enabled"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
      <topic
            label="The activity org.eclipse.javaDevelopment is enabled"
            href="/org.eclipse.ua.tests/data/help/context/simple_page.html">
      </topic>
   </context>
</contexts>